### PR TITLE
Users and posts

### PR DESCRIPTION
**Pull request by lekegitrepo**
Milestone 3: Users & posts
Created user (using devise) and post model. Created association between them.

### CODE REVIEW

**Comment by lekegitrepo** [comment]
Please, besides the standard code review, I'll like any professional suggestions that can help improve my approach.

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      flash[:warning] = 'Unable to update post, please try again'
+      render 'edit'
+    end
+  end
+
+  def destroy
+    @post.destroy
+    flash[:success] = 'Post successful deleted'
+    redirect_to root_path
+  end
```
Comment (Torres-ssf) [inline app/controllers/posts_controller.rb]: - [ ] When I tried to delete a created post I got this error: 
Screenshot from 2019-12-06 11-46-45

**Comment by lekegitrepo** [comment]
I've made all the required corrections(fix the delete bug and create a link to show all the users).

File: app/views/shared/_post.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="form-group">
+  <%= form_for (@my_post) do |f| %>
```
Comment (efrapp) [inline app/views/shared/_post.html.erb]: There is a bug here when I try to update the post with empty content. It is always good practice to test the forms with empty fields. Check the error here #4.

Comment (lekegitrepo) [reply app/views/shared/_post.html.erb]: I've fixed the update bug.

**Comment by lekegitrepo** [comment]
I've fixed the bug and I'll make the optional changes later.

**Pull request merged**